monitor/grafana/zeebe.json

```diff
@@ -1,83 +1,4 @@
 {
-  "__inputs": [
```

Comment:
:question: Could you please verify if removing this is ok?

Reply:
I imported the file to a new grafana dashboard and everything seems to be working 👍